libraries/AP_DroneCAN/AP_DroneCAN.cpp

```diff
@@ -1066,7 +1066,7 @@ void AP_DroneCAN::notify_state_send()
 #endif // HAL_BUILD_AP_PERIPH
 
     msg.aux_data_type = ARDUPILOT_INDICATION_NOTIFYSTATE_VEHICLE_YAW_EARTH_CENTIDEGREES;
-    uint16_t yaw_cd = (uint16_t)(360.0f - AP::ahrs().get_yaw_deg())*100.0f;
+    const uint16_t yaw_cd = (uint16_t)(AP::ahrs().get_yaw_deg()*100.0f);
```

Comment:
I think given the answer from @bugobliterator we should add a comment in the code, plus add a comment in the DSDL file explaining the orientation convention